### DISCUSSION TITLE
Merge conflict resolved, but PR shows as unresolved

### DISCUSSION THREAD

**Discussion by stryker-sk** [community/community#57457] [Pull Requests]
### Select Topic Area

Bug

### Body

I have resolved a merge conflict on my PR but it still shows as unresolved. I push a different change locally to try trigger an update, but it also no longer shows the change on the PR and I cannot merge.

HELP! I am blocked!

**Comment by basilevs** [comment]
I'm working on https://github.com/eclipse/lsp4e/pull/688 - at the moment it does not see latest commit from my branch, and computes the delta from the days old commit in the origin.

Overall, three commits from involved branches are not taken into account.

UPDATE: now it sees my latest commit (after 10 minutes delay), but comparison base is still incorrect.
UPDATE2: situation has completely resolved itself without any action on my side. So recommendation is probably to wait for an hour and see if that helps with your PR.

**Comment by shpingalet007** [comment]
Woah, seems like I have the same thing right now. I force pushed the branch on our repo and I see the next: PR is not updated, but the branch yes...

Pull request commits - https://github.com/pocketnetteam/pocketnet.gui/pull/1050/commits
Branch commits - https://github.com/pocketnetteam/pocketnet.gui/compare/fix2905

**Update:** Okay, the PR was synced after some 10 minutes. Anyway, that's not normal. Never happened before.
